pcsx2/DebugTools/DebugInterface.cpp

```diff
@@ -348,9 +348,10 @@ int R5900DebugInterface::getRegisterCount(int cat)
 	case EECAT_CP0:
 	case EECAT_FPR:
 	case EECAT_FCR:
-	case EECAT_VU0F:
 	case EECAT_VU0I:
 		return 32;
+	case EECAT_VU0F:
+		return 33;  // 32 + ACC
```

Comment:
I'd prefer if EECAT_VU0F was not moved. To keep things consistent with the other switch cases below.

Reply:
If it wasn't moved, it'd be changed to something like
```
case EECAT_CP0:
case EECAT_FPR:
case EECAT_FCR:
  return 32;
case EECAT_VU0F:
  return 33;
case EECAT_VU0I:
  return 32;
```
as the top cases rely on fallthrough. Restoring the order is OK if that's fine though.

Reply:
I'm fine with the change if it works properly.